xep-0198.xml

```diff
@@ -409,6 +425,13 @@
 </failed>
 ]]></example>
   <p>Stream management errors SHOULD be considered recoverable; however, misuse of stream management MAY result in termination of the stream.</p>
+  <p>When a remote entity acknowledges that it has handled a number of stanzas that is higher than the amount of stanzas that it was sent (by sending an 'h' value that is to high), the local entity SHOULD generate a stream error and close the stream:</p>
+  <example caption='Server closes old stream that was replaced by new stream'><![CDATA[
+<stream:error>
```

Comment:
As it was pointed it <conflict/> may not be the best choice. I'm also still missing the stream management specific stream error.

```
<stream:error>
      <undefined-condition
          xmlns='urn:ietf:params:xml:ns:xmpp-streams'/>
      <text xml:lang='en' xmlns='urn:ietf:params:xml:ns:xmpp-streams'>
        You acknowledged 42 stanzas while I only send you 12 so far.
      </text>
      <reported-handled-count-to-high xmlns='urn:xmpp:sm:3'/>
</stream:error>
```

Reply:
Same copy/paste error. This should've gone elsewhere.